.eslintrc

```diff
@@ -28,6 +29,16 @@
             "MethodDefinition": true,
             "ClassDeclaration": true
         }
-    }]
+    }],
+    "prettier/prettier": [
+      "error",
+      {
+        "trailingComma": "es5",
```

Comment:
Is it not an issue to set trailingComma to es5? Especially with hound's configuration